docs/pages/docs/handbook/dev.mdx

```diff
@@ -0,0 +1,97 @@
+# Development tasks in a Monorepo
+
+The vast majority of dev workflows look like this:
+
+1. Open a repository
+2. Run a `dev` task while they develop
+3. At the end of the day, shut down the `dev` task and close the repository.
+
+`dev` will likely be the most frequently run task in your repository, so getting it right is important.
+
+## Types of `dev` tasks
+
+`dev` tasks come in many shapes and sizes:
+
+1. Running a local development server for a web app
+1. Running [`nodemon`](https://www.npmjs.com/package/nodemon) to re-run a backend process every time code changes.
+1. Running tests in `--watch` mode
+
+## Setup with Turborepo
+
+You should specify your dev task like this in your `turbo.json`.
+
+```json filename="turbo.json"
+{
+  "pipeline": {
+    "dev": {
+      "cache": false
+    }
+  }
+}
+```
+
+Since `dev` tasks don't produce outputs, `outputs` is empty. Dev tasks are also unique in that you _never_ want to [cache](/docs/core-concepts/caching) them, so we set `cache` as `false`.
```

Comment:
```suggestion
Since `dev` tasks don't produce outputs, `outputs` is empty. Dev tasks are also unique in that you rarely want to [cache](/docs/core-concepts/caching) them, so we set `cache` as `false`.
```